pkg/api/configmanagement/register.go

```diff
@@ -57,12 +57,16 @@ const (
 	// RGControllerNamespace is the namespace used for the resource-group controller
 	RGControllerNamespace = "resource-group-system"
 
+	// MonitoringNamespace is the namespace used for Config Sync monitoring
+	MonitoringNamespace = "config-management-monitoring"
```

Comment:
The same variable is already declared in `pkg/metrics/otel.go`. We can consolidate them.

Reply:
Using metrics.MonitoringNamespace would cause an import cycle.

Reply:
That would work, changed.